fitbenchmarking/fitbenchmarking
# Adds placeholders for tests

#### Description of Work
 
Fixes #413 

Adds test placeholders for the `core` and `results_processing` modules.
 

#### Testing Instructions

1. Check that there are test files for all `core` and `results_processing`  modules

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
